core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -1197,11 +1209,11 @@ protected void tick() {
                 isInWorldBorderWarningArea = false;
             }
 
-
-            for (Tickable entity : entityCache.getTickableEntities()) {
-                entity.tick();
+            if (!isFrozen()) {
```

Comment:
Can just be a call to the variable instead of the getter.

Reply:
Fixed in latest commit.